trustfall_core/src/ir/mod.rs

```diff
@@ -365,6 +391,10 @@ where
         }
     }
 
+    /// The operation name as a `str`
+    ///
+    /// Note that these are the same as would be given to a GraphQL `op`
+    /// argumetn.
```

Comment:
Let's just merge these? The return type is in the function signature so I don't think we need to restate it.
```suggestion
    /// The operation name, as it would have appeared in the `@filter` directive `op` argument.
```